packages/react-scripts/backpack-addons/sriEnabled.js

```diff
@@ -1,6 +1,6 @@
 "use strict";
 
-const SubresourceIntegrityPlugin = require("webpack-subresource-integrity");
+const { SubresourceIntegrityPlugin } = require("webpack-subresource-integrity");
```

Comment:
Migrate v1 to v5 use new import syntax, see [here](https://github.com/waysact/webpack-subresource-integrity/blob/next/MIGRATE-v1-to-v5.md#migrate-from-version-1x-to-version-5x).